index.html

```diff
@@ -17,9 +17,11 @@
     </div>
 
     <div id="game-board">
-      <canvas id="canvas" width="500" height="700"></canvas>
+      <canvas id="canvas" width="500" height="700" style="background-image: url(/images/road.png); background-size: cover;"></canvas>
```

Comment:
Another way to do that is creating the image inside of Game's constructor.

```
const backgroundImage = new Image();
backgroundImage.src = "../images/road.png"
this.imgRoad = backgroundImage;
```
And then drawing it inside of the update function

`this.ctx.drawImage(this.imgRoad, 0, 0, canvas.width, canvas.height)`

So the road image is only going to show up when you click the start button!